src/client/components/start.js

```diff
@@ -0,0 +1,36 @@
+"use strict";
+
+import React, { Component } from "react";
+import styled from "styled-components";
+import Button from '@material-ui/core/Button';
+import { Link } from 'react-router-dom';
+
+/*************************************************************************/
+
+const StartBase = styled.div`
+  display: flex;
+  justify-content: center;
+  grid-area: main;
+`;
+
+export class Start extends Component {
```

Comment:
Start is confusing, is this where you end up once you login? Does it take over functioning as the landing page? Why do we need a start and a landing page?

Reply:
Landing was just a dummy page. It will be deleted in the next commit.

Reply:
All good